### ISSUE TITLE
Add a good Romanian extension 

### ISSUE THREAD

**Issue by Luca312** [tachiyomiorg/tachiyomi-extensions#16051]
### Source name

Animekage 

### Source link

https://ak554.anime-kage.eu/

### Source language

Romanian

### Other details

18+/NSFW = FALSE

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by CrepeTF** [comment]
The site uses videos...
![bruh](https://user-images.githubusercontent.com/70870719/231871634-5b0bd53c-2450-409c-acb7-944384d1c256.png)

**Comment by scb261** [comment]
There seems to be manga, but everything is behind login

**Comment by Luca312** [comment]
Yeah sorry it has animes but are manga to will have a change of becoming a extension?
